events/src/main/res/layout/fragment_event_details.xml

```diff
@@ -114,12 +114,12 @@
             <Button
                 android:id="@+id/registerToday"
                 style="@style/DetailButton"
-                android:layout_width="wrap_content"
+                android:layout_width="match_parent"
```

Comment:
`match_parent` is frowned upon in `ConstraintLayout`. Were you having issues here?

Reply:
might make sense to change to `0dp` which is what they recommend.